src/docs/03-Code Guidelines/05-Browser-Support.md

```diff
@@ -1,12 +1,23 @@
 All contents in YouSee DNA is build for and tested in the browsers below. Any YouSee site or web application must be tested in the same browsers.
 
 ## Desktop
-- Internet Explorer 11
-- Chrome (2 latest versions) for Windows and MacOS
-- Firefox (2 latest versions) for Windows and MacOS
-- Safari (2 latest versions) for MacOS
-- Edge (2 latest versions) for Windows
-
-## Mobile/Tablet
-- Chrome (2 latest versions) for iOS and Android
-- Safari (2 latest versions) for iOS
+- Chrome (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Firefox (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for MacOS <sup>[[2]](#ref2)</sup>
+- Edge (3 latest versions) for Windows <sup>[[1]](#ref1)</sup>
+- Internet Explorer 11 for Windows <sup>[[4]](#ref4)</sup>
+
+## Phone & Tablet
+- Chrome (3 latest versions) for Android and iOS <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for iOS <sup>[[2]](#ref2)</sup>
+- Samsung Internet (latest version) for Android <sup>[[3]](#ref3)</sup>
+
+## Notes
+Latest version cover only the major release version, patches and minor updates is not taken into consideration.
+
+<ol>
+    <li id="ref1"></a>Chrome, Firefox and Edge has a short regular release cycle, support for 2 prior version is necessarily.</li>

```

Comment:
@vercell 
There's still a closing `</a>` element here ;)

Reply:
Embarrassing! it's now fixed.